DEPS

```diff
@@ -65,6 +65,33 @@ hooks = [
     'pattern': 'src/electron/package.json',
     'name': 'electron_npm_deps'
   },
+  {
+    'action': [
+      'python',
+      '-c',
+      'import os; os.chdir("src"); os.chdir("electron"); os.system("git submodule update --init --recursive");',
+    ],
+    'pattern': 'src/electron',
+    'name': 'electron_submodules'
+  },
+  {
+    'action': [
+      'python',
+      '-c',
+      'import os; os.chdir("src"); os.chdir("electron"); os.chdir("vendor"); os.chdir("boto"); os.system("python setup.py build");',
+    ],
+    'pattern': 'src/electron',
+    'name': 'setup_boto',
+  },
+  {
+    'action': [
+      'python',
+      '-c',
+      'import os; os.chdir("src"); os.chdir("electron"); os.chdir("vendor"); os.chdir("requests"); os.system("python setup.py build");',
```

Comment:
@nornagon 
Would it make sense to move `boto` and `requests` to DEPS and download them only when releasing Electron?

Reply:
@alexeykuzmin I have a local commit on my work laptop of moving these two to `DEPS` but still putting them in `electron/vendor` so that paths and python `import` statements still work.  It doesn't actually make anything better by doing that though so I didn't end up pushing it up (we still need these manual hooks to run `setup.py`.  Is there a different advantage to putting them in `DEPS`?

Reply:
should these just be installed into the docker container?

Reply:
@nornagon That wouldn't help on macOS or Windows